render.go

```diff
@@ -60,8 +60,8 @@ type Options struct {
 	Layout string
 	// Extensions to parse template files from. Defaults to [".tmpl"].
 	Extensions []string
-	// Funcs is a slice of FuncMaps to apply to the template upon compilation. This is useful for helper functions. Defaults to [].
-	Funcs []template.FuncMap
+	// Funcs is a slice of FuncMaps to apply to the template upon compilation. This is useful for helper functions. Defaults to empty map.
+	Funcs template.FuncMap
```

Comment:
Hmmm, so this is the only part that I'm worried about. If we change this, it will break for existing users of the package. Can we achieve the same result (adding template funcs as render options), but leaving this as it was?

Reply:
Yeah, this would break existing code.
I will fix that.